pkg/volume/csi/csi_test.go

```diff
@@ -562,6 +565,7 @@ func TestCSI_VolumeAll(t *testing.T) {
 				}
 				csiDetacher := volDetacher.(*csiAttacher)
 				csiDetacher.csiClient = csiClient
+				<-waitForAttachDone
```

Comment:
if waitForAttachDone has been closed, the code will skip. And also means that Attach has been executed. :)

Reply:
https://github.com/golang/go/blob/9214677e7df1e6130249bc83d721130b00d829c4/src/runtime/chan.go#L509-L518

Reply:
Done :0

Reply:
Well done! 👍